Intermittent Integration Test Failure: test_nimbus_preview_flag

Seen here:

https://app.circleci.com/pipelines/github/mozilla/experimenter/54136/workflows/8f2efb87-ce1e-4a13-a7d4-fa829327964b/jobs/307117

```py

selenium = <selenium.webdriver.firefox.webdriver.WebDriver (session="e6285bb9-c626-4a74-ba99-d2b9e7f9bcee")>
experiment_url = 'https://nginx/nimbus/test_nimbus_preview_flagdemo_app5deb'
create_experiment = <function create_experiment.<locals>._create_experiment at 0x749512306660>
kinto_client = <nimbus.kinto.client.KintoClient object at 0x749511219220>
base_url = 'https://nginx/nimbus/'
experiment_name = 'test_nimbus_preview_flag[DEMO_APP]5deb'
demo_app = <nimbus.pages.demo_app.frontend.DemoAppPage object at 0x7495114e3890>

    @pytest.mark.cirrus_enrollment
    def test_nimbus_preview_flag(
        selenium,
        experiment_url,
        create_experiment,
        kinto_client,
        base_url,
        experiment_name,
        demo_app,
    ):
        create_experiment(
            selenium,
            reference_branch_value=demo_app.REFERENCE_BRANCH_VALUE,
            treatment_branch_value=demo_app.TREATMENT_BRANCH_VALUE,
        ).launch_to_preview()
    
        SummaryPage(selenium, experiment_url).open().wait_for_preview_status()
    
        navigate_to(selenium)
        selenium.refresh()
        assert demo_app.wait_for_result_text(
            [demo_app.DEFAULT_NOT_ENROLLED_TEXT]
        ).is_displayed()
    
        # Enable nimbus preview flag
        demo_app.fill_and_send_form_data(
            demo_app.CLIENT_1["client_id"],
            demo_app.CLIENT_1["context"],
            nimbus_preview=True,
        )
    
        displayed_text = demo_app.wait_for_result_text(
            [demo_app.CONTROL_BRANCH, demo_app.TREATMENT_BRANCH]
        ).text
        assert displayed_text in [demo_app.CONTROL_BRANCH, demo_app.TREATMENT_BRANCH]
    
        navigate_to(selenium)
        selenium.refresh()
        assert demo_app.wait_for_result_text(
            [demo_app.DEFAULT_NOT_ENROLLED_TEXT]
        ).is_displayed()
    
        # Not using nimbus preview flag
        demo_app.fill_and_send_form_data(
            demo_app.CLIENT_2["client_id"], demo_app.CLIENT_2["context"]
        )
    
>       assert demo_app.wait_for_result_text([demo_app.DEFAULT_WICKED_TEXT]).is_displayed()

experimenter/tests/integration/nimbus/test_cirrus_integration.py:302: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
experimenter/tests/integration/nimbus/pages/demo_app/frontend.py:41: in wait_for_result_text
    return self.wait_for_and_find_element(
experimenter/tests/integration/nimbus/pages/base.py:53: in wait_for_and_find_element
    self.wait_for_locator((strategy, locator), description, refresh=refresh)
experimenter/tests/integration/nimbus/pages/base.py:43: in wait_for_locator
    self.wait_with_refresh(locator, message=message)
experimenter/tests/integration/nimbus/pages/base.py:35: in wait_with_refresh
    self.wait.until(_wait_for_it, message=message)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <selenium.webdriver.support.wait.WebDriverWait (session="e6285bb9-c626-4a74-ba99-d2b9e7f9bcee")>
method = <function Base.wait_with_refresh.<locals>._wait_for_it at 0x7495111c0540>
message = 'Demo app frontend: could not find Result Text'

    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = "") -> T:
        """Wait until the method returns a value that is not False.
    
        Calls the method provided with the driver as an argument until the
        return value does not evaluate to ``False``.
    
        Parameters:
        -----------
        method: callable(WebDriver)
            - A callable object that takes a WebDriver instance as an argument.
    
        message: str
            - Optional message for :exc:`TimeoutException`
    
        Return:
        -------
        object: T
            - The result of the last call to `method`
    
        Raises:
        -------
        TimeoutException
            - If 'method' does not return a truthy value within the WebDriverWait
            object's timeout
    
        Example:
        --------
        >>> from selenium.webdriver.common.by import By
        >>> from selenium.webdriver.support.ui import WebDriverWait
        >>> from selenium.webdriver.support import expected_conditions as EC
    
        # Wait until an element is visible on the page
        >>> wait = WebDriverWait(driver, 10)
        >>> element = wait.until(EC.visibility_of_element_located((By.ID, "exampleId")))
        >>> print(element.text)
        """
        screen = None
        stacktrace = None
    
        end_time = time.monotonic() + self._timeout
        while True:
            try:
                value = method(self._driver)
                if value:
                    return value
            except self._ignored_exceptions as exc:
                screen = getattr(exc, "screen", None)
                stacktrace = getattr(exc, "stacktrace", None)
            if time.monotonic() > end_time:
                break
            time.sleep(self._poll)
>       raise TimeoutException(message, screen, stacktrace)
E       selenium.common.exceptions.TimeoutException: Message: Demo app frontend: could not find Result Text

/opt/venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:146: TimeoutException
```

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-5428)

--- a/experimenter/tests/integration/nimbus/pages/base.py
+++ b/experimenter/tests/integration/nimbus/pages/base.py
@@ -10,7 +10,7 @@ class Base(Page):
     """Base page."""
 
     def __init__(self, selenium, base_url, **kwargs):
-        super().__init__(selenium, base_url, timeout=120, **kwargs)
+        super().__init__(selenium, base_url, timeout=300, **kwargs)
 
     def wait_for_page_to_load(self):
         self.wait.until(EC.presence_of_element_located(self._page_wait_locator))

